LetsDoStuff.Domain/LdsContext.cs

```diff
@@ -7,10 +7,27 @@ public class LdsContext : DbContext
     {
         public DbSet<User> Users { get; set; }
 
+        public DbSet<Activity> Activities { get; set; }
+
+        public DbSet<Tag> Tags { get; set; }
+
         public LdsContext(DbContextOptions<LdsContext> options)
             : base(options)
         {
             Database.EnsureCreated();
         }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            optionsBuilder.UseSqlServer(@"Server=(localdb)\mssqllocaldb;Database=relationsdb;Trusted_Connection=True;");
+        }
```

Comment:
We are using the dependency injection, so we don't need this. See the https://metanit.com/sharp/aspnet5/6.1.php